tests/Makefile

```diff
@@ -0,0 +1,29 @@
+all: lint unit_tests
+
+lint:
```

Comment:
I've changed this to use prospector and renamed the rules.

Do you want me to copy the full pylint and prospector config from the instance manager?  It will likely get out of sync soon.